docs/en/index.md

```diff
@@ -1,66 +1,83 @@
+This extension provides integration of [Symfony Console](https://github.com/symfony/console) into [Nette Framework](https://www.nette.org).
+
+It allows you to create command-line commands directly within your application. These commands can be used for recurring tasks, as cronjobs, maintenances, imports and/or big things like sending newsletters.
 
 
 Installation
 -----------
 
-The best way to install Kdyby/Console is using  [Composer](http://getcomposer.org/):
+Fastest way is to use [Composer](http://getcomposer.org/) - run following command in your project root:
 
 ```sh
 $ composer require kdyby/console
 ```
 
-You can enable the extension using your neon config.
+Minimal configuration
+---------------------
 
-```yml
+First register new extension in your `config.neon`
+
+```
 extensions:
 	console: Kdyby\Console\DI\ConsoleExtension
 ```
 
-
-Minimal configuration
----------------------
-
-This extension creates new configuration section `console`, the absolute minimal configuration might look like this
+This creates new configuration section `console`, the absolute minimal configuration might look like this:
 
 ```yml
 console:
 	url: http://www.kdyby.org
 ```
 
-The `url` key specifies reference url that allows you to generate urls using Nette `UI\Presenter` in CLI (which is not possible otherwise). Another useful key is `commands` where you can register new commands. Look at the [Extending](#extending) part.
+The `url` key specifies reference url, that allows you to generate urls using Nette `UI\Presenter` in CLI (which is not possible otherwise).
 
+Now, your nette installation is ready to run commands! Try it:
+
+```sh
+$ php www/index.php
+```
 
 Writing commands
 ----------------
 
-Commands are like controllers, but for Symfony Console. Example command might look like this
+Example command might look like this:
 
 ```php
 namespace App\Console;
 
+use App\Models;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class SendNewslettersCommand extends Command
 {
-	protected function configure()
+	/** @var Models\NewsletterSender */
+	private $newsletterSender;
+
+	/**
+	 * @param Models\NewsletterSender $sender
+	 */
+	protected function __construct(Models\NewsletterSender $sender)
 	{
-		$this->setName('app:newsletter')
-			->setDescription('Sends the newsletter');
+		parent::__construct('app:newsletter'); // <-- run with `php www/index.php app:newsletter`
+		$this->setDescription('Sends the newsletter');
+		$this->newsletterSender = $sender;
 	}
 
+	/**
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
 	protected function execute(InputInterface $input, OutputInterface $output)
 	{
-		$newsletterSender = $this->getHelper('container')->getByType('Models\NewsletterSender');
```

Comment:
this is definitely bad, but I really don't wanna encourage constructor injection in console commands